README.md

```diff
@@ -323,17 +335,20 @@ app.UseForFeature(featureName, appBuilder =>
 
 Creating a feature filter provides a way to enable features based on criteria that you define. To implement a feature filter, the `IFeatureFilter` interface must be implemented. `IFeatureFilter` has a single method named `EvaluateAsync`. When a feature specifies that it can be enabled for a feature filter, the `EvaluateAsync` method is called. If `EvaluateAsync` returns `true` it means the feature should be enabled.
 
-The following snippet demonstrates how to add a customized feature filter `MyCriteriaFilter`.
+The following snippet demonstrates how to add customized feature filters `MyFirstCriteriaFilter` and `MySecondCriteriaFilter`.
 
 ``` C#
 services.AddFeatureManagement()
-        .AddFeatureFilter<"MyCriteriaFilter">();
+        .AddFeatureFilter<MyFirstCriteriaFilter>()
+        .AddFeatureFilter<MySecondCriteriaFilter>();
 ```
 
 Feature filters are registered by calling `AddFeatureFilter<T>` on the `IFeatureManagementBuilder` returned from `AddFeatureManagement`. These feature filters have access to the services that exist within the service collection that was used to add feature flags. Dependency injection can be used to retrieve these services.
 
 **Note:** When filters are referenced in feature flag settings (e.g. appsettings.json), the _Filter_ part of the type name should be omitted. Please refer to the `Filter Alias Attribute` section below for more details.
 

```

Comment:
This is already clear from the description in the AddScopedFeatureManagement section.

Reply:
Removed.